### PR TITLE
Updated changelog

### PR DESCRIPTION
**Pull request by zsvoboda**
I have updated changelog for last few versions.

### CODE REVIEW

**Comment by yenkins-admin** [comment]
Merged build triggered.

**Comment by yenkins-admin** [comment]
Merged build started.

**Comment by yenkins-admin** [comment]
Merged build finished.

**Comment by yenkins-admin** [comment]
Test PASSed.
Refer to this link for build results: https://ci.intgdc.com/job/BIA-pull-request-builder/104/

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,43 @@
 # GoodData Ruby SDK Changelog
 
+
+## 0.6.20
+- iterators for retrieval of project, domain, and group objects
+- use query resource inlining for retrieving user filters
+- added support of GoodData PGP SSO
+- project creation token is not taken from the ~/.gooddata file
```
Comment (korczis) [inline CHANGELOG.md]: Will not be better something like this?

- project creation token is now taken from the ~/.gooddata file

**Comment by yenkins-admin** [comment]
Merged build triggered.

**Comment by yenkins-admin** [comment]
Merged build started.

**Comment by korczis** [comment]
ok to merge

**Comment by yenkins-admin** [comment]
Merged build finished.

**Comment by yenkins-admin** [comment]
Test PASSed.
Refer to this link for build results: https://ci.intgdc.com/job/BIA-pull-request-builder/105/

**Pull request merged**